physical/raft/bolt_64bit_test.go

```diff
@@ -43,3 +43,31 @@ func Test_BoltOptions(t *testing.T) {
 		})
 	}
 }
+
+// TestMmapFlags tests the getMmapFlags function, ensuring it returns the appropriate integer representing the desired mmap flag.
+func TestMmapFlags(t *testing.T) {
+	t.Parallel()
```

Comment:
Because this test is setting env vars, it shouldn't be marked as parallel.

Reply:
Ohhh nice catch!